azure/scope/cluster_test.go

```diff
@@ -154,7 +154,6 @@ func TestAPIServerHost(t *testing.T) {
 				Namespace: "default",
 			},
 		}
-		cluster.Default()
```

Comment:
what changed here?

Reply:
See:

- https://github.com/kubernetes-sigs/cluster-api/pull/6225

In particular:

- https://github.com/kubernetes-sigs/cluster-api/pull/6225/files#diff-3dd0f28c0a5df6e110b3304d8ea006b42cd84a5c0a1ef305d3c1dab323924a68L55